lib/charms/nrpe_exporter/v0/nrpe_exporter.py

```diff
@@ -485,13 +489,17 @@ def _generate_alert(self, relation, cmd, id, unit) -> dict:
             },
         }
 
-    def _generate_prometheus_job(self, relation, unit, cmd, exporter_address, id) -> dict:
+    def _generate_prometheus_job(
+        self, relation, unit, cmd, exporter_address, id, nagios_host_context
```

Comment:
Add a comment about the new `nagios_host_context`?
In the future it would be handy to see right away what kind of 3rd party dep we have here.

Reply:
Done!